Ports/.port_include.sh

```diff
@@ -22,9 +22,15 @@ enable_ccache() {
     if command -v ccache &>/dev/null; then
         ccache_tooldir="${SERENITY_BUILD_DIR}/ccache"
         mkdir -p "$ccache_tooldir"
-        for tool in gcc g++ c++; do
-            ln -sf "$(command -v ccache)" "${ccache_tooldir}/${SERENITY_ARCH}-pc-serenity-${tool}"
-        done
+        if [ "$SERENITY_TOOLCHAIN" = "Clang" ]; then
+            for tool in clang clang++; do
```

Comment:
hmm. The previous version symlinked `c++` to ccache, but not `cc` ? 

Was this for host compile jobs being cached? or for target builds? If it's for host builds, probably makes sense to do it regardless, and add cc as well?

Reply:
`Toolchain/Local/$SERENITY_ARCH/bin/$SERENITY_ARCH-pc-serenity-cc` doesn't exist; only `c++` does.

Reply:
hmm werid. if it's intended for target builds, maybe we should link cc and c++ to clang/clang++ and `<arch>-pc-serenity-gcc/c++` in a smarter way?

Reply:
No ports should rely on calling `cc`/`cxx`, especially since we need to pass extra arguments down to `clang`, so a simple symlink won't work.

I'm not sure why `c++` was originally added to the list of executables to be wrapped by ccache. Should I remove it?